nepalingo-web/src/components/Flashcard.tsx

```diff
@@ -3,8 +3,16 @@ import Card from "./Card";
 import Button from "./Button";
 import useDictionary from "../hooks/useDictionary";
 import { generate } from "random-words";
+import ReactGA from "react-ga4";
 
 const Flashcard: React.FC = () => {
+  ReactGA.event({
+    category: "flash cards",
+    action: "Click",
+    value: 99, // optional, must be a number
+    nonInteraction: true, // optional, true/false
+    transport: "xhr", // optional, beacon/xhr/image
+  });
```

Comment:
This might run more than once per click due to the way react re-renders. It might be better to put this in the `handleFlip` function, but for now just approving as is.